<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>battery1Control</class>
 <widget class="QWidget" name="battery1Control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>battery1Control</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>BATTERY CONTROL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_op_state">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>192</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Operating Mode:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="minimum">
    <number>-3000</number>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QLabel" name="batt1_control_mode">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>Monitor Bus</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="batt1_apply_current_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>353</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="minimum">
    <number>-3000</number>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_control">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Control</string>
   </property>
  </widget>
  <widget class="QLabel" name="batt1_operating_state">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="batt1_set_ref_voltage">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>340</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>380</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
  </widget>
  <widget class="QLabel" name="batt1_ref_voltage">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="batt1_apply_voltage_pb">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="batt1_set_ref_current">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>340</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="batt1_switch_control_mode_pb">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>Switch
 Control Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ref_current">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>149</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Reference Current (mA): </string>
   </property>
  </widget>
  <widget class="QLabel" name="batt1_ref_current">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_status_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ref_voltage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>179</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Reference Voltage (V): </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_set_ref_voltage">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Set Ref Voltage:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="dial_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>353</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="maximum">
    <number>400</number>
   </property>
   <property name="value">
    <number>380</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="batt1_on_off_indicator">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_set_ref_current">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Set Ref Current:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="batt1_on_off_pb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
  <zorder>label_set_ref_voltage</zorder>
  <zorder>label_title</zorder>
  <zorder>label_op_state</zorder>
  <zorder>spinBox</zorder>
  <zorder>batt1_control_mode</zorder>
  <zorder>batt1_apply_current_pb</zorder>
  <zorder>dial</zorder>
  <zorder>label_control</zorder>
  <zorder>batt1_operating_state</zorder>
  <zorder>batt1_set_ref_voltage</zorder>
  <zorder>spinBox_2</zorder>
  <zorder>batt1_ref_voltage</zorder>
  <zorder>batt1_apply_voltage_pb</zorder>
  <zorder>batt1_set_ref_current</zorder>
  <zorder>batt1_switch_control_mode_pb</zorder>
  <zorder>label_ref_current</zorder>
  <zorder>batt1_ref_current</zorder>
  <zorder>label_status_2</zorder>
  <zorder>label_ref_voltage</zorder>
  <zorder>dial_2</zorder>
  <zorder>batt1_on_off_indicator</zorder>
  <zorder>label_set_ref_current</zorder>
  <zorder>batt1_on_off_pb</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>batt1_set_ref_voltage</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>batt1_set_ref_voltage</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>batt1_set_ref_current</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>batt1_set_ref_current</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
